src/algorithm/convexhull.rs

```diff
@@ -0,0 +1,174 @@
+use std::cmp::Ordering;
+use num_traits::Float;
+use types::{Point, Polygon, LineString};
+use std::collections::BTreeSet;
+
+impl<T> Eq for Point<T> where T: Float {}
```

Comment:
> `BTreeSet` needs `Ord`

Ah right, forgot about this. What you have is fine for now. Can you just add a comment next to the `Ord` implementation stating that it's needed for `BTreeSet`?